### PR TITLE
Trimorphic Number in Python #1736

### PR DESCRIPTION
**Pull request by U-c0de**
Completed issue : Trimorphic number in Python

### CODE REVIEW

**Comment by U-c0de** [comment]
Completed issue : Trimorphic number in Python #1736

**Comment by U-c0de** [comment]
Please Check --- @rish-singhal @yasharth291 @raksha009 @rudrakshi99 @plazzy99 @OjusWiZard @tarun26091999 @Rohan27s

**Comment by U-c0de** [comment]
@Rohan27s @rish-singhal  check this

**Review by Rohan27s — CHANGES_REQUESTED** [review]
Update readme first

**Comment by U-c0de** [comment]
@Rohan27s  Done

File: Code/Python/TrimorphicNumber.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+else:
+   print("The given is not number is Trimorphic")
+
+'''    TEST CASES
+    
+    1)Input: 
+            Enter the Number
+            20
+    2)Output:
+            The given number is Trimorphic
```
Comment (Rohan27s) [inline Code/Python/TrimorphicNumber.py:33]: fix this, 20 is not a trimorphic num

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Comment by U-c0de** [comment]
Done @Rohan27s

**Review by Rohan27s — APPROVED** [review]

**Pull request merged**